schematicexport.py

```diff
@@ -185,7 +185,7 @@ def _update_schematic7(self, path):
                 f.write(line + "\n")
         self.logger.info("Added LCSC's to %s (maybe?)", path)
 
-    def _update_schematic8(self, path):
+    def _update_schematic(self, path):
```

Comment:
can we make this _update_schematic8 for consistency and because we've got kicad 9 here and likely want to do v9 specific things.

Reply:
I wanted to invert the logic and keep the variant compatible with latest versions. When the versions 8/9 won't share the code anymore, we can split it into separated functions.